mmdet3d/structures/bbox_3d/base_box3d.py

```diff
@@ -463,17 +463,11 @@ def overlaps(cls, boxes1, boxes2, mode='iou'):
         # height overlap
         overlaps_h = cls.height_overlaps(boxes1, boxes2)
 
-        # Restrict the min values of W and H to avoid memory overflow in
-        # ``box_iou_rotated``.
-        boxes1_bev, boxes2_bev = boxes1.bev, boxes2.bev
```

Comment:
we should not modify this back. these codes aim to avoid overflow.
We could also add the PR link ( https://github.com/open-mmlab/mmdetection3d/pull/2134) in the code comments to remind future modification